set.mm

```diff
@@ -122142,8 +122142,16 @@ ordinal natural numbers (finite integers starting at 0), so that proof
   2p1e3 $p |- ( 2 + 1 ) = 3 $=
     ( c3 c2 c1 caddc co df-3 eqcomi ) ABCDEFG $.
 
-  $( 1 + 2 = 3.  (Contributed by David A. Wheeler, 8-Dec-2018.) $)
+  $( 1 + 2 = 3.  (Contributed by David A. Wheeler, 8-Dec-2018.)  Reduce
+     dependencies on axioms.  (Revised by Steven Nguyen, 12-Dec-2022.) $)
   1p2e3 $p |- ( 1 + 2 ) = 3 $=
+    ( c1 c2 caddc co c3 oveq2i ax-1cn addassi 1p1e2 oveq1i 2p1e3 eqtri 3eqtr2i
+    df-2 ) ABCDAAACDZCDOACDZEBOACNFAAAGGGHPBACDEOBACIJKLM $.
+
+  $( Obsolete version of ~ 1p2e3 as of 12-Dec-2022. 1 + 2 = 3.  (Contributed by
+     David A. Wheeler, 8-Dec-2018.)  (New usage is discouraged.)
+     (Proof modification is discouraged.) $)
+  1p2e3OLD $p |- ( 1 + 2 ) = 3 $=
     ( c2 c1 c3 2cn ax-1cn 2p1e3 addcomli ) ABCDEFG $.
```

Comment:
That axiom dependency reduction looks great.  Once it is on the website, I'll have a closer look.  Maybe, given that the OLD proof is much shorter, we will keep it as ALT ?

Reply:
I agree with @benjub that the old proof should be kept as ALT theorem (OLD theorems will be removed after one year...).